# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/michiel/blackbox/gcc33-support.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN -x '*~' blackbox-0.65.0/src/Window.cc blackbox-0.65.0-gcc-3.3.1/src/Window.cc
--- blackbox-0.65.0/src/Window.cc	2002-09-17 17:23:35.000000000 +0200
+++ blackbox-0.65.0-gcc-3.3.1/src/Window.cc	2003-08-14 22:22:57.000000000 +0200
@@ -44,6 +44,12 @@
 #endif // HAVE_STDLIB_H
 }
 
+#if defined(__GNUC__)
+#if __GNUC__ == 3
+#include <cassert>
+#endif
+#endif
+
 #include "i18n.hh"
 #include "blackbox.hh"
 #include "GCCache.hh"
